talk-proposals/SGibson-CogX20-BinderHub.md

```diff
@@ -0,0 +1,111 @@
+
+mybinder.org is both a public service and a proof of concept of Project Binder's technologies.
+This site serves over 140,000 sessions per week and combines resources from four entities in the US and Europe, including The Alan Turing Institute.
+We support a range of programming languages and analysts from a range of research domains, and even facilitate a number of remote workshops and tutorials.
+The flexibility of mybinder.org in allowing users to configure their environments provides scope to support a broad range of domains.
+See the Natural Language Processing course powered by mybinder.org (https://spacy.io/) and a Machine Learning course for explainable AI (https://github.com/GitiHubi/courseML).
+
+During the Covid-19 pandemic, mybinder.org has seen a significant increase in usage correlating with the beginning of lockdown and many researchers began working and collaborating remotely.
+It is quickly becoming the "go to" tool for sharing results and facilitating remote teaching in this new landscape of research culture.
+
+Dr Sarah Gibson is a member of the mybinder.org operating team and is personally responsible for maintaining the Binder infrastructure deployed at the Turing.
+During this talk, she will introduce Binder (the service), BinderHub (the technological infrastructure) and mybinder.org (a public instance of a Binder service, free for anyone to use) and demonstrate how it can be utilised to share reproducible computational environments and analyses.
+
+## Descriptors
+
+_Please select all descriptors that best apply to your talk._
+
+- [ ] Technical / specialist
+- [x] Big picture
+- [x] Detailed
+- [ ] Humorous
+- [ ] Serious
+- [ ] Retrospective
+- [x] Prospective
+- [ ] General / accessible
+
+## Domain Application
+
+_Please select up to two domains that best apply to your research and talk proposal._
+
+- [ ] Natural world
+- [ ] Security
+- [x] Engineering
+- [ ] Health
+- [x] Society & Culture
+- [ ] Economics
+
+## Methodological Application
+
+_Please select which methodological problem areas apply to your research and talk._
+
+- [ ] Finding ways to automate the lengthy process spent preparing data for analysis
+- [ ] Preventing misuse of data-driven technologies and artificial intelligence
+- [ ] Using and analysing data in order to confidently infer causal relationships
+- [x] Empowering citizens to access, understand and exploit the world's data
+- [ ] Improving the ways in which we collect and analyse data
+- [x] Effectively and efficiently convey information to varying audiences
+- [ ] Merging multiple sources of data
+- [ ] Using data while protecting privacy
+- [ ] Merging human knowledge with data-driven machine learning and AI systems
+- [ ] Eradicating failures within interacting systems
+- [ ] Optimising algorithmic and system performance for bigger and more complex datasets
+- [ ] Building on the most effective methods across data science to produce the next gen of techniques
+
+## Relevance
+
```

Comment:
Restructure Relevance section to be more around "why would a CogX attendee come to my talk".
Try to weave in tracks: ed tech rev, leadership, next gen infra.